docs/api.md

```diff
@@ -1100,6 +1114,56 @@ This endpoint supports retrieving extended metadata; provide a header `X-Extende
 + Response 403 (application/json)
     + Attributes (Error 403)
 
+# Group Datasets
+
+_(introduced: version 2022.3)_
+
+Dataset is a collection of entities. Dataset schema is created using form XML, you can see [ODK XForm](https://getodk.github.io/xforms-spec) specification for defining dataset in form XML. 
+
+Entities are added to dataset when a submission of the form that defines the  dataset is **approved**.
+
+### Related APIs:
+
+- [Link dataset to a form attachment](/reference/project-management/project-assignments)
+- [Get related datasets of a form](#)
+
+
+## Datasets [GET /projects/{projectId}/datasets]
+
+Returns the list of datasets in a project
+
++ Parameters
+    + projectId: `16` (number, required) - The numeric ID of the Project
+
++ Response 200 (application/json)
+    This is the standard response
+
+    + Attributes (array[Dataset])
+
++ Response 403 (application/json)
+    + Attributes (Error 403)
+
+## Download dataset [GET /projects/{projectId}/datasets/{name}/download]
```

Comment:
Done in https://github.com/getodk/central-backend/pull/689

Reply:
I've actually ended up thinking that it should be &hellip;/datasets/{name}/entities.csv. That's very similar to what we do for submissions: &hellip;/forms/{xmlFormId}/submissions.csv(.zip). Just as forms and submissions are different resources, I think datasets and entities should be treated as separate resources. I think it makes the most sense to think of what's being downloaded here as entities, not a dataset record. If we were to add a REST endpoint for entities, it would be &hellip;/datasets/{name}/entities, so I think &hellip;/datasets/{name}/entities.csv makes sense as a way to download those entities in a CSV format. I know we specifically forbad periods in dataset names in order to facilitate an endpoint at &;hellip/datasets/{name}.csv, but ultimately, I think &hellip;/datasets/{name}/entities.csv makes the most sense.

Reply:
Right, right, {name}.csv is for the OpenRosa Form Attachment endpoint. Agreed that the link I posted supports what you said.

Reply:
updated in doc as well